File: Szhchem.Data/Models/Collections/CalculatedProperty.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Szhchem.Data
{
    public class CalculatedProperty
    {
        [BsonIgnore]
        public long PropertyMolID { get; set; }
        public int ProgramCode { get; set; }
        public int PropertyCode { get; set; }
        public string PropertyText { get; set; }
    }
}

File: molfinder/ClientApp/src/models/CalculatedProperty.ts
export default class CalculatedProperty {
    public propertyCode: number;
    public programCode: number;
    public propertyText: string;
    public propertyMolID: bigint;

    constructor(propertyCode: number, programCode: number, propertyText: string, propertyMolID: bigint) {
        this.propertyCode = propertyCode;
        this.programCode = programCode;
        this.propertyText = propertyText;
        this.propertyMolID = propertyMolID;
    }
}